fenix/app/src/main/res/values/strings.xml

```diff
@@ -274,7 +274,7 @@
     <!-- Onboarding home screen sync popup dialog message, shown on top of Recent Synced Tabs in the Jump back in section. -->
     <string name="sync_cfr_message">Your tabs are syncing! Pick up where you left off on your other device.</string>
     <!-- Content description (not visible, for screen readers etc.): Close button for the home onboarding dialog -->
-    <string name="onboarding_home_content_description_close_button">Close</string>
+    <string name="onboarding_home_content_description_close_button" moz:RemovedIn="112" tools:ignore="UnusedResources">Close</string>
```

Comment:
I now see that we ended up using a 'common' content description for all these Close buttons. In the past, I have attempted to simplify the duplication of the multiple common strings we had such as Ok, Cancel, etc. However, this isn't quite right from a localization point of view. We generally would rather duplicate the string and provide context for each of their different usage. 

This was the most recent discussion because a contributor also wanted to de-duplicate the strings to have a common localized strings. https://github.com/mozilla-mobile/fenix/pull/27813#issuecomment-1316080412

Reply:
Ah I see, yeah the intention was to use the common strings for a common component, but seeing it has been an issue and has been discussed, it makes sense to use specific strings. Thanks for sharing the discussions, I've updated it!